acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/transaction/PantheonWeb3j.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.tests.acceptance.dsl.transaction;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+
+import tech.pegasys.pantheon.ethereum.core.Address;
+import tech.pegasys.pantheon.ethereum.core.Hash;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ScheduledExecutorService;
+
+import org.web3j.protocol.Web3jService;
+import org.web3j.protocol.core.JsonRpc2_0Web3j;
+import org.web3j.protocol.core.Request;
+import org.web3j.protocol.core.Response;
+
+public class PantheonWeb3j extends JsonRpc2_0Web3j {
```

Comment:
Is there a reason for us to keep using the Web3j?

Reply:
No, I think we should just use PantheonWeb3j as it also includes the all the standard web3j methods as well as our pantheon rpc methods. I've also changed the Transaction interface so it uses the PantheonWeb3j instead of Web3j.

Reply:
Does `PantheonNode` still reference `Web3j` internally?

Reply:
I've changed PantheonNode so it only uses PantheonWeb3j internally now